macos/Classes/PurchasesFlutterPlugin.h

```diff
@@ -0,0 +1 @@
+../../ios/Classes/PurchasesFlutterPlugin.h
```

Comment:
were these links done automatically?

Reply:
I made this link to avoid having to copy paste these files every time we update the iOS part of the plugin